CDCgov/cdc-open-viz
# [DEV-9291] Change mobile breakpoint for legend wrap

## Testing Steps

Create a chart, notice that the legend is forced to the bottom at `576px` instead of `768px`. This lets us use right-aligned legends in the center column of DFE if desired.

## Self Review

- I have added testing steps for reviewers
- I have commented my code, particularly in hard-to-understand areas
- My changes generate no new warnings
- New and existing unit tests are passing
